src/CarbonAware.WebApi/src/Controllers/CarbonAwareController.cs

```diff
@@ -204,6 +204,58 @@ public IActionResult BatchForecastData(IEnumerable<EmissionsForecastBatchDTO> re
         return Ok(result);
     }
 
+    /// <summary>
+    /// Retrieves the actual data that occurred between the time boundaries and calculates the carbon intensity that occurred during that window. 
+    /// </summary>
+    /// <remarks>
+    ///  This endpoint is useful for reporting the measured carbon intensity for a specific time period in a specific location.
+    /// </remarks>
+    /// <param name="location"> the location in </param>
+    /// <param name="jobStartTime"> the time at which the job and thus carbon usage starts </param>
+    /// <param name="jobEndTIme"> the time at which the job and thus carbon usage ends </param>
+    /// <returns> a single object that contains the location, time boundaries and carbon intensity value </returns>
+    /// <response code="200">Returns the a single object that contains the info about the request and the marginal carbon intensity </response>
+    /// <response code="400">Returned if any of the requested items are invalid</response>
+    /// <response code="500">Internal server error</response>
+    [Produces("application/json")]
+    [ProducesResponseType(StatusCodes.Status200OK, Type = typeof(IEnumerable<>))]
+    [ProducesResponseType(StatusCodes.Status400BadRequest, Type = typeof(ValidationProblemDetails))]
+    [ProducesResponseType(StatusCodes.Status500InternalServerError, Type = typeof(ValidationProblemDetails))]
+    [HttpGet("actual-carbon-intensity")]
```

Comment:
The term "actual" data is confusing to me as such a top-line descriptor.  I know we have been using it internally, but my assumption as an API user is that the underlying data is "actual" data unless otherwise specified.

Given these two endpoints using the same data source what is the difference?
`GET /emissions/byLocation?location=eastus&time=2022-07-21T12:00:00Z&toTime=2022-07-21T12:15:00Z`
`GET /emissions/actual-carbon-intensity?location=eastus&jobStartTime=2022-07-21T12:00:00Z&jobEndTIme=2022-07-21T12:15:00Z`

One could reasonably argue that the three EmissionsData objects returned by the first request better represents the "actual carbon intensity".

I think it may be more accurate to use keyword `average` in the naming and drop the word `actual`

Reply:
that makes sense to me, I changed it to average since the user would assume its based on the actual data